### PR TITLE
Vlimpers regulations rights

### PR DESCRIPTION
**Pull request by koenmetsu**

### CODE REVIEW

File: src/OrganisationRegistry/Handling/Authorization/AuthenticationResult.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+namespace OrganisationRegistry.Handling.Authorization
+{
+    public class AuthenticationResult
```
Comment (W0dan) [inline src/OrganisationRegistry/Handling/Authorization/AuthenticationResult.cs]: authorization result ?

File: test/OrganisationRegistry.UnitTests/Organisation/AddOrganisationParent/WhenAddingAParentWithOverlappingValidity.cs [hunk 2]
```diff
@@ -48,10 +49,11 @@ protected override IEnumerable<IEvent> Given()
             _validTo = _dateTimeProviderStub.Today.AddDays(2);
             _organisationId = Guid.NewGuid();
             _organisationParentId = Guid.NewGuid();
+            _ovoNumber = "OVO000012345";
 
             return new List<IEvent>
             {
-                new OrganisationCreated(_organisationId, "Kind en Gezin", "OVO000012345", "K&G", Article.None, "Kindjes en gezinnetjes", new List<Purpose>(), false, null, null, null, null),
+                new OrganisationCreated(_organisationId, "Kind en Gezin", _ovoNumber, "K&G", Article.None, "Kindjes en gezinnetjes", new List<Purpose>(), false, null, null, null, null),
```
Comment (W0dan) [inline test/OrganisationRegistry.UnitTests/Organisation/AddOrganisationParent/WhenAddingAParentWithOverlappingValidity.cs:56]: -> builder ?

File: test/OrganisationRegistry.UnitTests/Organisation/AddOrganisationParent/WhenAddingAnOrganisationParent.cs [hunk 3]
```diff
@@ -45,10 +46,11 @@ protected override IEnumerable<IEvent> Given()
             _validTo = _dateTimeProviderStub.Today.AddDays(2);
             _organisationId = Guid.NewGuid();
             _organisationParentId = Guid.NewGuid();
+            _ovoNumber = "OVO000012345";
 
             return new List<IEvent>
             {
-                new OrganisationCreated(_organisationId, "Kind en Gezin", "OVO000012345", "K&G", Article.None, "Kindjes en gezinnetjes", new List<Purpose>(), false, null, null, null, null),
```
Comment (W0dan) [inline test/OrganisationRegistry.UnitTests/Organisation/AddOrganisationParent/WhenAddingAnOrganisationParent.cs:51]: -> builder

File: test/OrganisationRegistry.UnitTests/Organisation/UpdateOrganisationParent/WhenChangingAnInactiveOrganisationParentBackAndForth.cs [hunk 4]
```diff
@@ -28,6 +28,7 @@ public class WhenChangingAnInactiveOrganisationParentBackAndForth : Specificatio
         private DateTime _validFrom;
 
         private static readonly DateTime Yesterday = DateTime.Now.AddDays(-1);
```
Comment (W0dan) [inline test/OrganisationRegistry.UnitTests/Organisation/UpdateOrganisationParent/WhenChangingAnInactiveOrganisationParentBackAndForth.cs:30]: don't reference DateTime.Now  in tests .. -> preferably fixed datetimes

**Review by W0dan — COMMENTED** [review]
commentjes ...

**Comment by koenmetsu** [comment]
@W0dan all comments resolved.

**Review by W0dan — APPROVED** [review]

**Pull request merged**